api/routes/comments.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const router = express.Router();
+
+const {
+  getAllComments,
+  addComment,
+  removeComment,
+  updateComment,
+} = require("../controllers/comments.js");
+
+router.get("/", async (req, res) => {
+  res.send(await getAllComments());
```

Comment:
Better keep await operations in separate line

Reply:
Changed